.github/workflows/release.yml

```diff
@@ -84,3 +84,65 @@ jobs:
         with:
           draft: true
           files: "${{ env.FILES }}"
+
+  rocksdb-x86_64-windows:    
+    needs: ["goreleaser"]
+    runs-on: windows-latest
+    environment: release    
+    steps:
+      - name: get source
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: setup msys2
+        uses: msys2/setup-msys2@v2
+        with:
+          msystem: MINGW64          
+          update: false
+          install: git msys2-devel base-devel mingw-w64-x86_64-gcc mingw-w64-x86_64-rocksdb  mingw-w64-x86_64-go  mingw-w64-x86_64-jq  mingw-w64-x86_64-dlfcn        
+      - name: get rocksdb & build
+        env:
+          PLATFORM: Windows_x86_64
+        shell: msys2 {0}
+        run: |
+          uname -a
+          export GOROOT=/mingw64/lib/go
+          export GOPATH=$HOME/go
+          export PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+          export COSMOS_BUILD_OPTIONS=rocksdb   
+          export CGO_CFLAGS="-I/mingw64/include/rocksdb" 
+          export CGO_LDFLAGS="-L/mingw64/lib -lrocksdb -lstdc++ -lm -lz -lbz2 -lsnappy -llz4 -lzstd" 
+          echo $PATH
+          go version
+          wget https://mirror.msys2.org/mingw/mingw64/mingw-w64-x86_64-rocksdb-6.26.1-1-any.pkg.tar.zst
+          pacman -U mingw-w64-x86_64-rocksdb-6.26.1-1-any.pkg.tar.zst --noconfirm
+          make      
```

Comment:
I think you need `COSMOS_BUILD_OPTIONS=rocksdb make` to make a rocksdb build.

Reply:
it's declared before, for clarity,
I'll modify above